application/app/models/project.rb

```diff
@@ -79,22 +82,27 @@ def to_s
     to_json
   end
 
+  def destroy
+    project_path = self.class.projects_directory(id)
+    FileUtils.rm_rf(project_path)
+  end
```

Comment:
- we should control at the model or controller level when we destroy a project and this project is active. It should be deactivated before destroying 

- we should control at the model level when we try to destroy a project and there are ongoing downloads taking place (if we asume that a project is a bundle of downloaded files. In the implementation I propose, files should be added to the project as project_files once they are downloaded, and what we understand as download collection should be a download selection bundle)

Reply:
Yes, we need to add validation on the delete to avoid those edge cases. Will raise an issue to implement later.
Not sure I understand the second point, but we can discuss in the brainstorming.

Reply:
(2) maybe soft delete is a good approach and it would solve the issue (2) I reported on the comment above

Reply:
yes - will raise the issue to review and improve delete projects. At the moment is simply to start the fuctionality and learn the implications of deleting as well as to get feedback from stakeholders

Reply:
issue created:
https://github.com/hmdc/DevOpsProjects/issues/1739